src/book/02-system/02-subsystems/06-behaviour.mdx

```diff
@@ -47,19 +55,30 @@ The [**fake/keyboardwalk** role](https://github.com/NUbots/NUbots/blob/main/role
 
 The [PS3 walk module](https://github.com/NUbots/NUbots/tree/main/module/behaviour/strategy/PS3Walk) is an interface for **testing** and **demonstrating** the walk and other functionalities using a **PS3 controller**.
 
-The module connects to the PS3 controller using bluetooth (or USB) and reacts based on joystick or button input.
+The module connects to the PS3 controller using USB and reacts based on joystick or button input.
```

Comment:
Is there a problem wth bluetooth? We had it going early last year with the PS3 walk.

Reply:
It didn't work when we last tried it 2 weeks ago. I'll have another look on Friday.